boot/zephyr/prj-p256.conf

```diff
@@ -11,8 +11,8 @@ CONFIG_MBEDTLS_CFG_FILE="config-asn1.h"
 # CONFIG_TINYCRYPT_ECC_DSA is not set
 # CONFIG_TINYCRYPT_SHA256 is not set
 
-### mbedTLS wants a heap
-CONFIG_HEAP_MEM_POOL_SIZE=16384
+### mbedTLS has its own heap
+# CONFIG_HEAP_MEM_POOL_SIZE is not set
```

Comment:
Not familiar with this API -- is it necessary to have an "is not set" line, or can this be removed from the prj.conf completely?

Reply:
I wasn't quite sure what the best way was. Leaving it out causes it to be "unchanged" from any default. The "is not set" will explicitly unset it, even if there is a default. I did this to try and make sure to remove any heap if it were to become set in Zephyr's default.